solutions/community-deploy/cloud/AWS/multi-node.tf

```diff
@@ -0,0 +1,164 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.27"
+    }
+  }
+  required_version = ">= 0.15.1"
+}
+
+provider "aws" {
+  profile = "default"
+  region  = var.region
+}
+resource "aws_default_vpc" "default" {}
+
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+resource "aws_default_subnet" "default" {
+  availability_zone = data.aws_availability_zones.available.names[0]
+}
+
+locals {
+  common_tags = {
+    "Terraform" = "true",
+    "CORTX"     = "true"
+  }
+}
+
+resource "aws_security_group" "cortx_deploy" {
+  name        = "cortx_deploy"
+  description = "Allow standard ssh, CORTX mangement ports inbound and everything else outbound."
+
+  ingress {
+    description = "SSH Acces"
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = [var.security_group_cidr]
+  }
+
+  ingress {
+    description = "CORTX UI Access"
+    from_port   = 443
+    to_port     = 443
+    protocol    = "tcp"
+    cidr_blocks = [var.security_group_cidr]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = local.common_tags
+}
+
+data "aws_ami" "centos" {
+  most_recent = true
+  owners      = ["125523088429"]
```

Comment:
why it is hardcoded?

Reply:
Yes. we can use this as a variable. @shailesh-vaidya should i go ahead and use this as variable ?

Reply:
Yes. Go ahead and use it as a variable.